org.eclipse.january.test/src/org/eclipse/january/dataset/LazyMathsTest.java

```diff
@@ -86,17 +83,15 @@ private void testMaxMultipleAxes(boolean allowMaths) throws Exception {
 			LazyMaths.setAllowDatasetMaths(true);
```

Comment:
Can you also refactor to remove the `LazyMaths.setAllowDatasetMaths()` and `LazyMaths.isAllowDatasetMaths`?  Then these private `tesMXXMultipleAxes()` methods here will have a parameter `useLazyDataset` and be `b4x4x4x4` versus `b4x4x4x4L`, etc.